### ISSUE TITLE
example use request for IndexedSetter methods.

### ISSUE THREAD

**Issue by tphyahoo** [ekmett/lens#565]
I've been tweaking the docs on the operator page at

https://github.com/ekmett/lens/wiki/Operators

As someone with recent exposure to lens, I think it would be good if every operator linked there had examples of use in the hackage.  This is the case for the left hand column of Control.Lens section but I hit my first confusion in "Control.Lens (Indexed)"

Could you add example usages in the haddock for iover at least, and perhaps other methods in this section there, in Control.Lens.Setter.

Specifically, I can't even figure out how to produce a value of the right type for iover (an IndexedSetter).

Thanks.

My hope is to make the operators page a friendlier jumping off point.

**Comment by tphyahoo** [comment]
It looks like there is reasonable documentation here:

http://ekmett.github.io/lens/Control-Lens-Indexed.html

I think this could be used to flesh out examples for indexed part of operators wiki page, and then for style points we could add haddock examples to the package.

**Comment by ekmett** [comment]
One downside to all of this new content there is that it has taken the operator chart from something you could visually skim to something where you can't "see the forest for the trees". When the descriptions were one-liners you could skim up and down the different columns to get a sense of how things related. 

Now it just looks like symbol-soup.

Not sure if there is a better way to go about it, maybe splitting things up or finding a way to make them collapsible?

**Comment by bennofs** [comment]
I'll try putting the examples on a new line and see how that looks.

**Comment by ekmett** [comment]
Thanks!

**Comment by bennofs** [comment]
I edited the wiki to make all examples be on a new line, but I have to admit that I still liked the old Operators page more. Adding examples to Operators leads to the Operators page becoming more of a bad replication of the Haddocks rather than the quick operator/non-operator lookup table that it used to be.

**Comment by ekmett** [comment]
What if we made a separate section at the bottom of the page with the examples from each operator group, and then put an [examples] link as a final column in each one.

Then we have a fixed size for the cells and the whole thing will collapse into a tight grid, and you can have a densely packed run of examples at the bottom with links directly there from the operators themselves?

**Comment by bennofs** [comment]
Yeah, that would work. Still, the operators already link to haddocks where there are examples, and those examples are compiled. Do we really wish to keep another set of examples for each combinator? And there's also already a wiki page Examples too.

**Comment by ekmett** [comment]
That might also make sense.

It seems this chart may be better served by reducing the number of examples and forcing folks to follow the links to the operators.

**Comment by ekmett** [comment]
This issue here was really about adding usage in the _haddock_ at first anyways, so these goals seem independent.

**Comment by ekmett** [comment]
Alright, I'll make the executive decision, lets tighten up the [Operators](https://github.com/ekmett/lens/wiki/Operators) page, and I'll take pull requests adding examples to the haddocks for the ones that are missing.

**Comment by ekmett** [comment]
I've cleaned up the operators page, but I'm leaving this open to remind us to add more examples to the haddocks.

There also appear to be a few dead links from the Operators page.

**Comment by tphyahoo** [comment]
ok, will work on incorporating appropriate examples to haddocks.
